docs/contributor/storage/configuration.md

```diff
@@ -62,7 +69,37 @@ device-classes:
 ```
 Caveats:
 - Specifying lvcreate-options is at your own risk and only provided as an escape hatch by the CSI plugin. 
-- Setting spare-gb to anything other than 0 is not recommended because it does not behave predictably and usually results in more space being allocated that expected.
+- Setting spare-gb to anything other than 0 is not recommended because it does not behave predictably and usually results
+in more space being allocated that expected.
+
+#### LVM Thin Volumes
+
+Advanced storage features such as volume cloning and snapshotting are only supported on thin volumes, and thus require an
+LVM thin-pool on the host and the appropriate LVMS and cluster configuration.  For information on creating a thin-pool,
+see [OpenShift documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/creating-and-managing-thin-provisioned-volumes_configuring-and-managing-logical-volumes).
+
+> VMs created via the [Getting Started](../../user/getting_started.md) instructions are provisioned with a single
+thin-pool automatically. 
```

Comment:
I'm not sure it's useful to cross-reference Getting Started document here just for this case, unless it's already referenced somewhere else.

Reply:
+1 I'll drop it.